### ISSUE TITLE
[development] Cannot parse file names with colons.

### ISSUE THREAD

**Issue by e-e-e** [samiz-dat/samiz-dat#21]
For example:
![screen shot 2017-11-27 at 10 14 39 pm](https://user-images.githubusercontent.com/12589522/33264177-9583fa5a-d3c0-11e7-82a7-d222861f19f1.png)

Throws this error:
```
"TypeError: Cannot read property 'name' of undefined"
found in
---> <DatBook> at 
```

Use this key `dat://12a32e4543f1a3777f1256280ee63353d47da1a4adb0e70903eee663078e7d5f` to access the library with plenty of examples.

This is only within the development branch.

**Comment by e-e-e** [comment]
More examples
![screen shot 2017-11-27 at 10 21 38 pm](https://user-images.githubusercontent.com/12589522/33264390-70dc800e-d3c1-11e7-9c80-bf915aedf817.png)

**Comment by e-e-e** [comment]
fixed in v0.2.0
